.gitpod.yml

```diff
@@ -11,5 +11,6 @@ tasks:
 - init: >
     mvn install
   command: >
-    cd phoenicis-javafx/ &&
-    mvn exec:java
+    mvn clean package &&
+    cd phoenicis-dist/src/scripts &&
+    sh phoenicis-create-package.sh
```

Comment:
This way you can only build the package but not run it to test your code changes (which should be the main purpose for gitpod).